### PR TITLE
BaSP-TG-115: Responsive component's

### PR DESCRIPTION
**Pull request by agusting19**
There has been a few changes in the aesthetic and every component is now responsive.

You should be able to see all the app in all kind of device (360x640 to 1440x).

### CODE REVIEW

File: src/Components/Shared/Table/table.module.css [hunk 1]
```diff
@@ -48,3 +53,70 @@
...
+  }
+  td:nth-of-type(3):before {
+    content: 'Phone:';
+  }
+  td:nth-of-type(4):before {
+    content: 'Email:';
+  }
+  td:nth-of-type(5):before {
+    content: 'Status:';
+  }
```
Comment (Javier-Cervera) [inline src/Components/Shared/Table/table.module.css:121]: Only takes into account the data of an employee
(Worked Hours, Client Name, Start Date, etc?)

File: src/Components/Admin/Projects/ProjectPage/projectPage.module.css [hunk 2]
```diff
@@ -1,48 +1,34 @@
...
+.btnContainer button {
+  margin: 20px 0px 20px 10px;
 }
 
 .box {
   display: flex;
   justify-content: center;
   flex-wrap: wrap;
-  width: 90%;
   margin: 10px;
```
Comment (Javier-Cervera) [inline src/Components/Admin/Projects/ProjectPage/projectPage.module.css:25]: On mobile --> `max-width: 90%` or `max-width: 90vw` could help with the border overflow

**Review by Javier-Cervera — COMMENTED** [review]
In `Components\Shared\ModalForm\modal-form.module.css`, on `.modal{ }` --> `max-width: 90vh;` could have a correct unit, like `vw`

In `Components\Admin\Projects\ProjectPage\MemberForm\memberForm.module.css`, on `.container{ }` --> `width: 20vw;` could be `width: unset;`

**Review by NicolasFontana — CHANGES_REQUESTED** [review]
When viewing the sidebar in mobile, the following happens:

image


It would be nice if it went to the bottom of everything

**Review by NicolasFontana — CHANGES_REQUESTED** [review]
When the text is very long and there is a line break, the text is no longer centered

image

**Review by NicolasFontana — APPROVED** [review]
Looks nice!

**Comment by LeonelListro** [comment]
# Test Results

Tested localy on: https://alexis-trackgenix-app-git-feature-tg-115-resp-2554a4-basp-m2022.vercel.app/home

## Test results: Failed ⚠️

**Bugs found:**

**320*480 (Smartphone)**

1- The side bar is too big when it's displayed.

image

2- The burger menu should be fixed on the screen (when you roll down de page the side bar go down too).

Sin título

3- In the login page there is no space between the main conteiner and the header and footer.

Sin título

4- In the employees admin table the delete button should be centered.

Sin título

5- In the projects admin table when you add new project, the modal is broken.

image

6- In the employee profile when you try to edit the profile, the modal is broken (do not show the title of the modal).

image

7- In the super admin home the main title should be centered.

image

8- In the super admin admin table the elements should be centered.

image


**768*1023 (Tablet)**

1- The side bar is too big when it's displayed.

image

2- The wrong password modal should be centered.

image

3- The suggestion of user should not be there.

Sin título


**1440*920 (Desktop)**

1- The suggestions of users should not be there.

image


**General bugs:**

1- When you try to add a new admin using an email address that is already in use the page crashes.

image

**Review by LeonelListro — CHANGES_REQUESTED** [review]
Test failed. Please refer to https://github.com/BaSP-m2022/alexis-trackgenix-app/pull/65#issuecomment-1194689381

**Review by LeonelListro — APPROVED** [review]
Well done Agustin and Team!

**Pull request merged**